plugins/node/opentelemetry-instrumentation-router/src/instrumentation.ts

```diff
@@ -28,19 +28,15 @@ import * as http from 'http';
 import type * as Router from 'router';
 
 import * as types from './internal-types';
-import { VERSION } from './version';
+import { PACKAGE_NAME, PACKAGE_VERSION } from './version';
 import * as constants from './constants';
 import * as utils from './utils';
 import AttributeNames from './enums/AttributeNames';
 import LayerType from './enums/LayerType';
 
 export default class RouterInstrumentation extends InstrumentationBase {
   constructor(config: InstrumentationConfig = {}) {
-    super(
-      `@opentelemetry/instrumentation-${constants.MODULE_NAME}`,
-      VERSION,
-      config
-    );
+    super(PACKAGE_NAME, PACKAGE_VERSION, config);
```

Comment:
As part of this change I think we could remove the `MODULE_NAME` constant from the `constants.ts` file. Then we can also rename in the `init()` there is another `constants.MODULE_NAME`.

Reply:
I totally agree, this pattern should be simplified and refactored to a common one across the repo 👍 

the `MODULE_NAME` export in `constants.ts` currently looks like this:

```js
export const MODULE_NAME = 'router';
```

And still being used in instrumentation [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/0bec64e2e9a73735c7cbecd7212f5be6ef575308/plugins/node/opentelemetry-instrumentation-router/src/instrumentation.ts#L50). We cannot utilize the new `PACKAGE_NAME` to remove it altogether at the moment.

This PR touches on the topic of "Instrumentation Package" and the `MODULE_NAME` we refer to is the "Instrumented Package". In order not to introduce too many things into one PR which can make it harder to review and merge in, I suggest we will target the instrumented package patterns in a follow-up PR, which might also include more cleanups and aligning across the codebase.

Reply:
Yeah makes sense to have a new PR for the additional cleanup 👍